app/components/UserLibraryList.vue

```diff
@@ -1,81 +1,107 @@
-  
-  const currentUser = await getCurrentUser();
- 
-  onMounted(async () => {
-    const { data: fetchedDocument } = await $fetch<ApiResponse<AnnoteDocument[]>>('/api/annote_documents');
-    if (fetchedDocument) {
-      annoteDocs.value = fetchedDocument;
-    }
-   
-  });
+const annoteDocs = ref<AnnoteDocument[] | null>(null);
+const { getCurrentUser } = useAuth();
+
+const currentUser = await getCurrentUser();
+
+onMounted(async () => {
+  const { data: fetchedDocument } = await $fetch<ApiResponse<AnnoteDocument[]>>(
+    "/api/annote_documents"
+  );
+  if (fetchedDocument) {
+    annoteDocs.value = fetchedDocument;
+  }
+});
 </script>
 
 <template>
-  <div class="mt-4">
+  <div class="w-2/3 mx-auto mt-4">
+    <h1 class="text-2xl">Library</h1>
     <!-- This search bar section -->
-    <div class="py-4 flex justify-end gap-x-2 pr-4">
-
-      <div class="searchTextField lightRoundedGreyBorder flex">
-        <div class="self-center mt-2">
+    <div class="flex justify-end py-4 pr-4 gap-x-2">
+      <div class="flex searchTextField lightRoundedGreyBorder">
+        <div class="self-center mt-2 ml-2">
           <Icon name="mdi:magnify" color="black" size="1.5rem" />
         </div>
-        <input type="text" placeholder="Search" class="searchInput w-full p-2 border border-black" />
+        <input
+          type="text"
+          placeholder="Search"
+          class="w-full p-2 border border-black searchInput"
+        />
       </div>
 
       <!-- Drop down search filter -->
-      <div class="lightRoundedGreyBorder content-center">
+      <div class="content-center lightRoundedGreyBorder">
         <select>
           <option value="createdAscending">
             <p>&#129031; Date Created Ascending</p>
           </option>
-          <option value="createdDescending">&#129029; Date Created Descending</option>
-          <option value="alphaAscending">&#129031; Alphabetical Ascending</option>
-          <option value="alphaDescending">&#129029; Alphabetical Descending</option>
+          <option value="createdDescending">
+            &#129029; Date Created Descending
+          </option>
+          <option value="alphaAscending">
+            &#129031; Alphabetical Ascending
+          </option>
+          <option value="alphaDescending">
+            &#129029; Alphabetical Descending
+          </option>
         </select>
       </div>
     </div>
+
+    <!-- This is the list of annote documents -->
     <ul>
-      <li class="border-t border-black pt-2 pb-2 pl-2" v-for="doc in annoteDocs" :key="doc.document_id">
-        <div class="flex justify-between">
+      <li
+        v-for="(doc, index) in annoteDocs"
+        :key="doc.document_id"
+        :class="[
+          'p-4 hover:custom-green duration-200',
+          index % 2 === 0 ? 'bg-gray-100' : 'bg-white',
+        ]"
+      >
+        <div class="flex items-center justify-between">
           <div>
-            <p class="truncatable-text">
+            <p>
```

Comment:
In the Figma there is a document icon on the left and next to the title for each doc? 
There's also the `visibility` and the number of `stickies` - is that something you can address in this PR?

Reply:
Updated this just now. Not sure how to access the number of stickies that currently exist in the doc though. Looks like I have to count the number of blocks with a cdx marker class?

Reply:
You can get all of the stickies by document_id and count them